src/main/java/lotto/constant/LottoRanking.java

```diff
@@ -0,0 +1,28 @@
+package lotto.constant;
+
+import java.util.Arrays;
+
+public enum LottoRanking {
+
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000),
+    ;
+
+    private final int ranking;
+    private final int award;
+
+    LottoRanking(int ranking, int award) {
+        this.ranking = ranking;
+        this.award = award;
+    }
+
+    public static int getAward(int ranking) {
+        LottoRanking rankingAward = Arrays.stream(LottoRanking.values())
+                .filter(value -> value.ranking == ranking)
+                .findFirst().orElseThrow(IllegalArgumentException::new);
```

Comment:
```suggestion
                .findFirst()
                .orElseThrow(() -> new IllegalArgumentException(String.format("%d에 해당하는 값을 찾을 수 없습니다.", ranking)));
```
메세지에 원인을 적어주시면 오류를 쉽게 파악할 수 있습니다ㅎㅎ